Test/spec/simd/simd_i16x8_arith2.wast

```diff
@@ -242,7 +242,7 @@
 (assert_invalid (module (func (result v128) (i16x8.max_s (i32.const 0) (f32.const 0.0)))) "type mismatch")
 (assert_invalid (module (func (result v128) (i16x8.max_u (i32.const 0) (f32.const 0.0)))) "type mismatch")
 
-;; Test operation with empty argument
+;; Argument empty
```

Comment:
Nit: the previous comment seems more descriptive to me.

Reply:
Since this is a small nit and the PR otherwise LGTM, I'll merge now and if you want you can change these comments in a follow up.

Reply:
Oh lol nevermind I didn't notice that this was in WAVM :joy:

Reply:
Thank you @tlively, I will send the PR soon.